hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,116 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+
+    def generate_outside(box, inside, multipliers):
+        """Generate test cases from interior points by adding lattice vectors."""
+        a = numpy.array([box[0], 0, 0])
+        b = numpy.array([box[1] * box[3], box[1], 0])
+        c = numpy.array([box[2] * box[4], box[2] * box[5], box[2]])
+        out = numpy.zeros((len(inside), len(multipliers), 3))
+        ins = numpy.zeros_like(out)
+        mults = numpy.zeros_like(out)
+        for i, point in enumerate(inside):
+            for j, f in enumerate(multipliers):
+                out[i, j, :] = point + a * f[0] + b * f[1] + c * f[2]
+                ins[i, j, :] = point
+                mults[i, j, :] = f
+        return out.reshape((-1, 3)), ins.reshape((-1, 3)), mults.reshape(
+            (-1, 3))
+
+    if s.communicator.rank == 0:
+        # multiples of lattice vectors to add to interior points to generate tests
+        multiples = [
+            [0, 0, 0],
+            [1, 0, 0],
+            [0, 1, 0],
+            [0, 0, 1],
+            [-1, 0, 0],
+            [0, -1, 0],
+            [0, 0, -1],
+            [-1, -1, -1],
+            [-5, 24, 13],
+            [3, -4, 5],
+            [3, 4, -5],
+            [100, 101, 102],
+            [-50, -50, 50],
+        ]
+
+        box = [1, 1, 1, 0, 0, 0]
+        inside = [[0, 0, 0], [-0.5, 0.0, -0.2], [0.0, 0.3, -0.1],
+                  [0.3, 0.2, -0.1], [-0.5, 0.2, -0.2]]
+        outs, ins, mults = generate_outside(box, inside, multiples)
+        s.configuration.box = box
+        s.particles.N = len(ins)
+        s.particles.position[:] = outs
+        s.wrap()
+        numpy.testing.assert_allclose(s.particles.position, ins, atol=1e-12)
+        numpy.testing.assert_allclose(s.particles.image, mults)
```

Comment:
You can use `assert_array_equal` here. The images are integers and do not need a tolerance check.